cli/azd/internal/repository/app_init.go

```diff
@@ -911,11 +911,18 @@ func (i *Initializer) addMavenBuildHook(
 	wrapperPathMap := map[string][]string{}
 
 	for _, prj := range detect.Services {
-		if prj.Language == appdetect.Java && prj.Options["parentPath"] != nil {
-			parentPath := prj.Options[appdetect.JavaProjectOptionMavenParentPath].(string)
-			posixMavenWrapperPath := prj.Options[appdetect.JavaProjectOptionPosixMavenWrapperPath].(string)
-			winMavenWrapperPath := prj.Options[appdetect.JavaProjectOptionWinMavenWrapperPath].(string)
-			wrapperPathMap[parentPath] = []string{posixMavenWrapperPath, winMavenWrapperPath}
+		if prj.Language == appdetect.Java {
+			if prj.Options["parentPath"] != nil {
```

Comment:
Use const variable instead of string here. `"parentPath"` -> `appdetect.JavaProjectOptionMavenParentPath`

Reply:
Updated.